lib/parsers/page.js

```diff
@@ -34,7 +34,7 @@ function Page(attrs) {
     }
   }
 
-  this.url = '/' + (this.path.indexOf(path.sep) >= 0 ? path.dirname(this.path) : '')
```

Comment:
Personally I prefer `/about` over `/about.html`. Hence I'd name the file as `about/index.html` rather than `about.html`. It seems we need a `if else` check here.